GA-worksheet.md

```diff
@@ -7,62 +7,62 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+The purpose of a header tag is for headers.
 2. What is a Navigation Bar?
+A list of links.
+3. What tag do we use to make a navigation bar? <ul  >
 
-3. What tag do we use to make a navigation bar? <      >
-
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
+4.  <ul> Stands for _unordered lists____________________
+    <li> Stands for _list item_____________________
 
 5. What does the <a> tag do?
-
+links another page.
 6. What would the line of HTML below do?
 
 ``` html
 <a href="http://www.google.com">Google</a>
 ```
-
+it would open your browser.
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
-
+lets you keep CSS in a external file and keeps it more organized.
 2. What is normalize.css?
-
+CSS file that makes sure CSS is rendered the same. 
 3. Why do we need to add a padding property to the `<ul>` tag?
-
+because there is padding there, it must be set to 0.
 4. What does the “inline” property do to a list of items?
-
+it puts all the information on the same line.
 5. What do the following padding values do?
 
     padding: 25px 50px 75px 100px;
 
-    * top padding is __px
-    * right padding is __px
-    * bottom padding is __px
-    * left padding is __px
+    * top padding is _25_px
+    * right padding is _50_px
+    * bottom padding is _75_px
+    * left padding is _100_px
 
     padding: 20px;
 
-    * top padding is __px
-    * right padding is __px
-    * bottom padding is __px
-    * left padding is __px
+    * top padding is _20_px
+    * right padding is 20__px
+    * bottom padding is _20_px
+    * left padding is _20_px
 
 6. What is the difference between padding and margin?
-
+padding is the space inside the border and margin is the opposite.
 7. What did border-radius do to the square border around the image?
-
+it makes it rounder.
 ##Project #2 Part #3: Responsive Design and Javascript
 
 1. Why is responsive design becoming a more important aspect to consider when making a website compared to ten years ago?
-
+So more people would be intrigued to go on the website.
```

Comment:
Our main goal is to make sure our websites look awesome on any device. If a website looks bad, it has a bad "user experience" and people might not want to use it. We didn't have to worry about this 10 years ago, because smart phones and tablets weren't available.